src/features/game/players/Players.tsx

```diff
@@ -1,31 +1,59 @@
-import React from "react";
+import React, { ReactElement } from "react";
 import { useSelector } from "react-redux";
 import {
   getActiveGamePlayers,
   getActiveGameSpectator,
   getActiveGameTurn,
 } from "../../../redux/games";
+import { getCurrentPlayerId } from "../../../redux/currentPlayer";
 import ActivePlayer from "./ActivePlayer";
 import Spectator from "./Spectator";
 
 const Players = () => {
   const players = useSelector(getActiveGamePlayers);
   const spectators = useSelector(getActiveGameSpectator);
+  const currentPlayer: string = useSelector(getCurrentPlayerId);
   const turn = useSelector(getActiveGameTurn);
+
+  const currentPlayerCard: ReactElement[] = [];
+  const restPlayers: ReactElement[] = [];
+
+  (() => {
+    players.forEach((each: string) => {
+      if (each === currentPlayer) {
+        currentPlayerCard.push(
+          <ActivePlayer
+            key={each}
+            playerId={each}
+            players={players}
+            turn={turn}
+          />
+        );
+      } else {
+        restPlayers.push(
+          <ActivePlayer
+            key={each}
+            playerId={each}
+            players={players}
+            turn={turn}
+          />
+        );
+      }
+    });
+  })();
+
   return (
     <div className="grid grid-rows-1 grid-cols-1">
       <div className="cols-span-1 rows-span-1 my-4">
-        Players
-        <div className="w-full flex flex-row">
-          {players &&
-            players.map((each: string) => (
-              <ActivePlayer
-                key={each}
-                playerId={each}
-                players={players}
-                turn={turn}
-              />
-            ))}
+        <div className="flex flex-row gap-1 justify-center">
+          <div className="flex flex-row items-center">
+            <div className="order-1"> {currentPlayerCard}</div>
+            <span
+              className="bg-black m-1 order-last"
+              style={{ height: "70%", width: `0.18rem`, margin: "1rem" }}
+            ></span>
+          </div>
+          <div className="w-full flex flex-row gap-1">{restPlayers}</div>
         </div>
       </div>
       {spectators.length > 0 && (
```

Comment:
Instead of spectators taking up so much space, we should just add the spectator count next to the game status something like the screenshot below.

However we should change the icon to an eye

![Screenshot 2022-12-15 at 1 30 24 PM](https://user-images.githubusercontent.com/4240966/207804738-6f9da1a5-5bbd-4794-9985-d0d0f49c733e.png)